python/lsst/obs/sdss/sdssNullIsr.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
```

Comment:
We used to use it for unit tests, but changed the rules due to the "SIP" issue on macOS. We still use it for scripts in the `bin` directory and perhaps in examples (though due to the "SIP" issue those won't run standalone on macOS). I'm not sure how this one got here; possibly a copy/paste error or typo.